models/request.js

```diff
@@ -17,20 +17,27 @@
  */
 
 module.exports = function(sequelize, DataTypes) {
-    return sequelize.define('Request', {
-	name: DataTypes.TEXT,
-	address: DataTypes.TEXT,
-        sms_raw_address: DataTypes.TEXT,
-	assigned_rc_region: DataTypes.TEXT,
-	city: DataTypes.TEXT,
-	state: DataTypes.TEXT,
-	zip: DataTypes.TEXT,
-        sms_raw_zip: DataTypes.TEXT,
-	phone: DataTypes.TEXT,
-        sms_raw_phone: DataTypes.TEXT,
-	email: DataTypes.TEXT,
-        source: DataTypes.TEXT,
-        serial: { type: DataTypes.TEXT, unique: true },
-        status: DataTypes.TEXT
-    })
+  var Request = sequelize.define('Request', {
+    name: DataTypes.TEXT,
+    address: DataTypes.TEXT,
+          sms_raw_address: DataTypes.TEXT,
+    assigned_rc_region: DataTypes.TEXT,
+    city: DataTypes.TEXT,
+    state: DataTypes.TEXT,
+    zip: DataTypes.TEXT,
+          sms_raw_zip: DataTypes.TEXT,
+    phone: DataTypes.TEXT,
+          sms_raw_phone: DataTypes.TEXT,
+    email: DataTypes.TEXT,
+          source: DataTypes.TEXT,
+          serial: { type: DataTypes.TEXT, unique: true },
+          status: DataTypes.TEXT
```

Comment:
Try to minimize these unrelated whitespace changes as much as possible.  It makes the diff confusing to read.  And I know the indentation wasn't consistent to begin with, but now it's even less so.  Can you consistify, please?

Reply:
Sorry about that! Part of the indentation change was because of the different structure for the models based off of Sequelize, but the indentation definitely needed cleaning. Fixed in the latest commit